packages/fx-core/src/component/driver/deploy/azure/impl/azureDeployImpl.ts

```diff
@@ -174,6 +174,34 @@ export abstract class AzureDeployImpl extends BaseDeployImpl {
       azureCredential,
       azureResource.subscriptionId
     ));
+    try {
+      const defaultScope = "https://management.azure.com/.default";
+      const token = await azureCredential.getToken(defaultScope);
+      if (token) {
+        this.logger?.info("Get AAD token successful. Upload zip package through AAD Auth mode.");
+        return {
+          headers: {
+            "Content-Type": "application/octet-stream",
+            "Cache-Control": "no-cache",
+            Authorization: `Bearer ${token.token}`,
+          },
+          maxContentLength: Infinity,
+          maxBodyLength: Infinity,
+          timeout: DeployConstant.DEPLOY_TIMEOUT_IN_MS,
+        };
+      } else {
```

Comment:
Since basic auth will be disabled, why don't we just delete the basic auth related code?
For example: just throw error if `token` is empty.

Reply:
This method has not been thoroughly tested; it is unclear if any accounts are unable to access the information.